src/common/interfaces/IGatewaySenders.sol

```diff
@@ -124,6 +124,13 @@ interface IHubMessageSender is ILocalCentrifugeId {
 
 /// @notice Interface for dispatch-only gateway
 interface ISpokeMessageSender is ILocalCentrifugeId {
+    struct UpdateData {
```

Comment:
Went back and forth on this, but overall felt this was the cleanest outcome:
- Struct is needed regardless, to avoid stack too deep issue in balance sheet
- These 4 params are shared between the 2 update methods, so it makes sense to re-use
- Putting it in `ISpokeMessageSender` makes the calls and events shorter, which overall makes the code easier to read IMO